hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/ServiceFactory.java

```diff
@@ -279,4 +282,21 @@ public int maxPendingCallsPerProcessor() {
     public boolean hasOrderedAsyncResponses() {
         return orderedAsyncResponses;
     }
+
+    public interface ServiceContext<S> {
+        int localIndex();
+
+        int globalIndex();
+
+        int jetMemberIndex();
```

Comment:
we use memberIndex in PS.Context

Reply:
In this higher-level API I thought this would be clearer.